integration/tests-common/integration-test-utils/src/main/java/org/wso2/esb/integration/common/utils/Utils.java

```diff
@@ -362,4 +368,17 @@ public static boolean isCarFileDeployed(String carFileName, ApplicationAdminClie
         }
         return isCarFileDeployed;
     }
+
+    public static void deployConfiguration(OMElement config, String location) {
+
+        try (OutputStream outputStream = new FileOutputStream(location)) {
+            config.serialize(outputStream);
+            config.serialize(System.out);
+            CarbonServerExtension.restartServer();
```

Comment:
Shall we pass a boolean whether to restart the server or not .. Then we can deploy multiple configs and restart after them . WDYT

Reply:
+1